content/first-order-logic/completeness/compactness.tex

```diff
@@ -149,7 +149,7 @@
 \tagprob{FOL}
 \begin{prob}
 In the standard model of arithmetic~$\Struct{N}$, there is no
-!!{element}~$k \in \Domain{N}$ which satisfies every formula~$\num{n}
+!!{element}~$n \in \Domain{N}$ which satisfies every formula~$\num{n}
```

Comment:
k is actually correct. A k which satisfies all \num{n} < x would have to be larger than all n. No such k exists in the standard model. Can you change that back and push?

Reply:
Yep, just changed it back and pushed! Thanks

Reply:
No thank you!